pkg/client/actionclient.go

```diff
@@ -100,13 +103,23 @@ func AppendInstallFailureUninstallOptions(opts ...UninstallOption) ActionClientG
 		return nil
 	}
 }
+
 func AppendUpgradeFailureRollbackOptions(opts ...RollbackOption) ActionClientGetterOption {
 	return func(getter *actionClientGetter) error {
 		getter.upgradeFailureRollbackOpts = append(getter.upgradeFailureRollbackOpts, opts...)
 		return nil
 	}
 }
 
+type PostRendererProvider func(rm meta.RESTMapper, kubeClient kube.Interface, obj client.Object) postrender.PostRenderer
```

Comment:
I don't like to call everything "Getter" but this would keep naming consistent across this package.

```suggestion
type PostRendererGetter func(rm meta.RESTMapper, kubeClient kube.Interface, obj client.Object) postrender.PostRenderer
```